src/akkudoktoreosserver/flask_server.py

```diff
@@ -132,9 +140,13 @@ def flask_gesamtlast():
 @app.route("/gesamtlast_simple", methods=["GET"])
 def flask_gesamtlast_simple():
```

Comment:
still a bit of DEnglish here..
is flask_totalload_simple() a better name? 
or how about flask_total_load_simple?

maybe this is not the best time to rename now, as it's in alignment with the endpoints' name... 
Once the endpoint will be renamed, this method shall be renamed accordingly.